lib/api/core/httpRouter/routeHandler.js

```diff
@@ -7,75 +28,75 @@ const Request = require('kuzzle-common-objects').Request;
  * containing the information gathered about
  * a requested route and the corresponding handler
  * to invoke
- * @constructor
  *
+ * @class RouteHandler
  * @param {string} url - parsed URL
  * @param {string} requestId
  * @param {object} query - HTTP request query parameters
  * @param {object} headers
  */
-function RouteHandler(url, query, requestId, headers) {
-  this.handler = null;
-  this.url = url;
-  this.data = {requestId};
-  this.request = null;
+class RouteHandler {
+  constructor(url, query, requestId, headers) {
+    this.handler = null;
+    this.url = url;
+    this.data = {requestId};
+    this.request = null;
 
-  Object.keys(headers).forEach(k => {
-    if (k.toLowerCase() !== 'authorization') {
-      this.data[k] = headers[k];
-    }
-    else if (headers[k].startsWith('Bearer ')) {
-      this.data.jwt = headers[k].substring('Bearer '.length);
-    }
-  });
+    Object.keys(headers).forEach(k => {
+      if (k.toLowerCase() === 'authorization' && headers[k].startsWith('Bearer ')) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 84 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)